docs/timeline/index.html

```diff
@@ -1480,6 +1480,42 @@ <h2 id="Events">Events</h2>
     </tr>
 
     <tr>
+      <td>mouseOver</td>
+      <td>
+        Passes a properties object as returned by the method <a href="#getEventProperties"><code>Timeline.getEventProperties(event)</code></a>.
+      </td>
+      <td>Fired when mouse over the Timeline.
+      </td>
+    </tr>
+
+    <tr>
+      <td>mouseDown</td>
+      <td>
+        Passes a properties object as returned by the method <a href="#getEventProperties"><code>Timeline.getEventProperties(event)</code></a>.
+      </td>
+      <td>Fired when mouse down inside the Timeline.
+      </td>
+    </tr>
+
+    <tr>
+      <td>mouseUp</td>
```

Comment:
I don't understand this event. I only managed to trigger this in the example by pressing the middle-mouse-button.